include/swift/Subsystems.h

```diff
@@ -232,28 +232,30 @@ namespace swift {
 
   /// Turn the given Swift module into either LLVM IR or native code
   /// and return the generated LLVM IR module.
-  std::unique_ptr<llvm::Module> performIRGeneration(IRGenOptions &Opts,
-                                                    ModuleDecl *M,
-                                                    SILModule *SILMod,
-                                                    StringRef ModuleName,
-                                                llvm::LLVMContext &LLVMContext);
+  std::unique_ptr<llvm::Module>
+  performIRGeneration(IRGenOptions &Opts,
+                      ModuleDecl *M,
+                      SILModule *SILMod,
+                      StringRef ModuleName,
+                      llvm::LLVMContext &LLVMContext);
 
   /// Turn the given Swift module into either LLVM IR or native code
   /// and return the generated LLVM IR module.
-  std::unique_ptr<llvm::Module> performIRGeneration(IRGenOptions &Opts,
-                                                    SourceFile &SF,
-                                                    SILModule *SILMod,
-                                                    StringRef ModuleName,
-                                                 llvm::LLVMContext &LLVMContext,
-                                                    unsigned StartElem = 0);
+  std::unique_ptr<llvm::Module>
```

Comment:
Can you see what clang-format does on this text? Try using git-clang-format. Many of us use that as our baseline of changes like this, so matching what clang-format does will reduce future changes by clang-format.